CODE_STYLE.md

```diff
@@ -89,7 +94,7 @@
   same include subdirectory as the .fh files.  See below for details.
 ## Fortran Example
 
-```Fortran ./example.F90 file
+```Fortran ./fms_public_example_example.F90 file
```

Comment:
The use of `public` here and in the sample code below isn't what's captured in lines 13-15 above where it simply says `fms_subfolder_`.  The inclusion of `public` implies we want everything to have `public` included in the naming scheme.  I'd also recommend not have the double example called out here.

Reply:
I updated with your suggestion.  Thank you.